CHANGELOG.md

```diff
@@ -27,7 +27,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 [1.1.0]: https://github.com/atlassian-labs/db-replica/compare/release-1.0.0...release-1.1.0
 
 ### Added
-- `AuroraPostgresLsnReplicaConsistency`
+- `KeyBasedThrottledCache`
+- `SuppliedCache<T>`
```

Comment:
I should be added in `unreleased` section... not in `1.1.0 - 2021-03-18` section